### ISSUE TITLE
Audio screen share is broken by 0.4.3 on Linux Mint 21.2

### ISSUE THREAD

**Issue by Stonley890** [Vencord/Vesktop#246]
Hello! I recently updated from Vesktop 0.4.2 to 0.4.3 and it seems that the audio sharing is no longer compatible. I find this odd, as 0.4.3 specifically is supposed to add compatibility.

Vesktop tells me I need glibc version 2.36 or higher (which I cannot install on Ubuntu 22.04/Linux Mint 21.2), but 0.4.2 seems to work as intended.
```
$ ldd --version
ldd (Ubuntu GLIBC 2.35-0ubuntu3.4) 2.35
Copyright (C) 2022 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```

**To Reproduce**
Using Linux Mint 21.2 Cinnamon, install Vesktop 0.4.3 and start a screen share.

**Workarounds**
Uninstall Vesktop 0.4.3 and install Vesktop 0.4.2

Below is my system information.
```
System:
  Kernel: 5.19.0-50-generic x86_64 bits: 64 compiler: N/A Desktop: Cinnamon 5.8.4 tk: GTK 3.24.33
    wm: muffin dm: LightDM Distro: Linux Mint 21.2 Victoria base: Ubuntu 22.04 jammy
Machine:
  Type: Desktop System: Gigabyte product: B660M DS3H DDR4 v: N/A serial: <superuser required>
  Mobo: Gigabyte model: B660M DS3H DDR4 v: x.x serial: <superuser required>
    UEFI: American Megatrends LLC. v: F5 date: 01/17/2022
Battery:
  Device-1: hidpp_battery_0 model: Logitech Wireless Mouse MX Master 2S serial: <filter>
    charge: 55% (should be ignored) status: Discharging
CPU:
  Info: 10-core (6-mt/4-st) model: 12th Gen Intel Core i5-12600K bits: 64 type: MST AMCP
    arch: Alder Lake rev: 2 cache: L1: 864 KiB L2: 9.5 MiB L3: 20 MiB
  Speed (MHz): avg: 2388 high: 3006 min/max: 800/4900:3600 cores: 1: 2532 2: 2305 3: 3006
    4: 2336 5: 2031 6: 2661 7: 2159 8: 2238 9: 2010 10: 2565 11: 2247 12: 2217 13: 2273 14: 2234
    15: 2915 16: 2489 bogomips: 117964
  Flags: avx avx2 ht lm nx pae sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx
Graphics:
  Device-1: Intel AlderLake-S GT1 vendor: Gigabyte driver: i915 v: kernel ports: active: none
    empty: DP-1, DP-2, DP-3, HDMI-A-1, HDMI-A-2, HDMI-A-3 bus-ID: 00:02.0 chip-ID: 8086:4680
  Device-2: NVIDIA GA106 [GeForce RTX 3060 Lite Hash Rate] vendor: PNY driver: nvidia
    v: 535.129.03 pcie: speed: 16 GT/s lanes: 16 ports: active: none off: DP-4,DP-5
    empty: DP-6,HDMI-A-4 bus-ID: 01:00.0 chip-ID: 10de:2504
  Display: x11 server: X.Org v: 1.21.1.4 driver: X: loaded: modesetting,nvidia
    unloaded: fbdev,nouveau,vesa gpu: i915,nvidia display-ID: :0 screens: 1
  Screen-1: 0 s-res: 4480x1440 s-dpi: 96
  Monitor-1: DP-0 pos: right res: 2560x1440 dpi: 109 diag: 686mm (27")
  Monitor-2: DP-2 pos: primary,left res: 1920x1080 dpi: 94 diag: 598mm (23.5")
  OpenGL: renderer: NVIDIA GeForce RTX 3060/PCIe/SSE2 v: 4.6.0 NVIDIA 535.129.03
    direct render: Yes
Audio:
  Device-1: Intel vendor: Gigabyte driver: snd_hda_intel v: kernel bus-ID: 00:1f.3
    chip-ID: 8086:7ad0
  Device-2: NVIDIA vendor: PNY driver: snd_hda_intel v: kernel pcie: speed: 16 GT/s lanes: 16
    bus-ID: 01:00.1 chip-ID: 10de:228e
  Device-3: Blue Microphones Yeti Stereo Microphone type: USB
    driver: hid-generic,snd-usb-audio,usbhid bus-ID: 1-5:3 chip-ID: b58e:9e84
  Sound Server-1: ALSA v: k5.19.0-50-generic running: yes
  Sound Server-2: PulseAudio v: 15.99.1 running: no
  Sound Server-3: PipeWire v: 0.3.48 running: yes
Network:
  Device-1: Realtek RTL8125 2.5GbE vendor: Gigabyte driver: r8169 v: kernel pcie: speed: 5 GT/s
    lanes: 1 port: 3000 bus-ID: 04:00.0 chip-ID: 10ec:8125
  IF: enp4s0 state: down mac: <filter>
  Device-2: Intel Wi-Fi 6 AX200 driver: iwlwifi v: kernel pcie: speed: 5 GT/s lanes: 1
    bus-ID: 05:00.0 chip-ID: 8086:2723
  IF: wlp5s0 state: up mac: <filter>
  IF-ID-1: virbr0 state: down mac: <filter>
Bluetooth:
  Device-1: Intel AX200 Bluetooth type: USB driver: btusb v: 0.8 bus-ID: 1-7.1:7
    chip-ID: 8087:0029
  Report: hciconfig ID: hci0 rfk-id: 0 state: up address: <filter> bt-v: 3.0 lmp-v: 5.2
    sub-v: 237e
Drives:
  Local Storage: total: 2.5 TiB used: 840.76 GiB (32.8%)
  ID-1: /dev/nvme0n1 vendor: Samsung model: SSD 970 EVO Plus 500GB size: 465.76 GiB
    speed: 31.6 Gb/s lanes: 4 serial: <filter> temp: 42.9 C
  ID-2: /dev/sda vendor: Toshiba model: MQ01ABD075 size: 698.64 GiB speed: 6.0 Gb/s
    serial: <filter>
  ID-3: /dev/sdb vendor: Western Digital model: WD5000AAKS-00V1A0 size: 465.76 GiB
    speed: 3.0 Gb/s serial: <filter>
  ID-4: /dev/sdc vendor: Western Digital model: WD1002FBYS-18A6B0 size: 931.51 GiB
    speed: 3.0 Gb/s serial: <filter>
Partition:
  ID-1: / size: 456.89 GiB used: 299.06 GiB (65.5%) fs: ext4 dev: /dev/nvme0n1p2
  ID-2: /boot/efi size: 511 MiB used: 6.1 MiB (1.2%) fs: vfat dev: /dev/nvme0n1p1
Swap:
  ID-1: swap-1 type: file size: 8 GiB used: 1024 KiB (0.0%) priority: -2 file: /swapfile
Sensors:
  System Temperatures: cpu: 27.8 C mobo: 16.8 C gpu: nvidia temp: 43 C
  Fan Speeds (RPM): N/A gpu: nvidia fan: 30%
Repos:
  Packages: 3620 apt: 3581 flatpak: 39
  No active apt repos in: /etc/apt/sources.list
  Active apt repos in: /etc/apt/sources.list.d/additional-repositories.list
    1: deb http: //security.debian.org/debian-security bookworm-security main
  Active apt repos in: /etc/apt/sources.list.d/brave-browser-release.list
    1: deb [signed-by=/usr/share/keyrings/brave-browser-archive-keyring.gpg] https: //brave-browser-apt-release.s3.brave.com/ stable main
  Active apt repos in: /etc/apt/sources.list.d/grapejuice.list
    1: deb [signed-by=/usr/share/keyrings/grapejuice-archive-keyring.gpg] https: //brinkervii.gitlab.io/grapejuice/repositories/debian/ universal main
  Active apt repos in: /etc/apt/sources.list.d/kdenlive-kdenlive-stable-jammy.list
    1: deb [arch=amd64 signed-by=/etc/apt/keyrings/kdenlive-kdenlive-stable-jammy.gpg] https: //ppa.launchpadcontent.net/kdenlive/kdenlive-stable/ubuntu jammy main
  Active apt repos in: /etc/apt/sources.list.d/microsoft-prod.list
    1: deb [arch=amd64,arm64,armhf] https: //packages.microsoft.com/ubuntu/22.04/prod jammy main
  Active apt repos in: /etc/apt/sources.list.d/minecraft-linux-pkg.list
    1: deb [arch=amd64,arm64] https: //minecraft-linux.github.io/pkg/deb jammy main
  Active apt repos in: /etc/apt/sources.list.d/mono-official-stable.list
    1: deb [signed-by=/usr/share/keyrings/mono-official-archive-keyring.gpg] https: //download.mono-project.com/repo/ubuntu stable-focal main
  No active apt repos in: /etc/apt/sources.list.d/nodesource.list
  Active apt repos in: /etc/apt/sources.list.d/obsproject-obs-studio-jammy.list
    1: deb [arch=amd64 signed-by=/etc/apt/keyrings/obsproject-obs-studio-jammy.gpg] https: //ppa.launchpadcontent.net/obsproject/obs-studio/ubuntu jammy main
  Active apt repos in: /etc/apt/sources.list.d/official-package-repositories.list
    1: deb https: //mirrors.ocf.berkeley.edu/linuxmint-packages victoria main upstream import backport
    2: deb http: //mirrors.ocf.berkeley.edu/ubuntu jammy main restricted universe multiverse
    3: deb http: //mirrors.ocf.berkeley.edu/ubuntu jammy-updates main restricted universe multiverse
    4: deb http: //mirrors.ocf.berkeley.edu/ubuntu jammy-backports main restricted universe multiverse
    5: deb http: //security.ubuntu.com/ubuntu/ jammy-security main restricted universe multiverse
  Active apt repos in: /etc/apt/sources.list.d/trebelnik-stefina-grub-customizer-jammy.list
    1: deb [signed-by=/etc/apt/keyrings/trebelnik-stefina-grub-customizer-jammy.gpg] https: //ppa.launchpadcontent.net/trebelnik-stefina/grub-customizer/ubuntu jammy main
  Active apt repos in: /etc/apt/sources.list.d/ubuntu-toolchain-r-test-jammy.list
    1: deb [signed-by=/etc/apt/keyrings/ubuntu-toolchain-r-test-jammy.gpg] https: //ppa.launchpadcontent.net/ubuntu-toolchain-r/test/ubuntu jammy main
  Active apt repos in: /etc/apt/sources.list.d/unityhub.list
    1: deb [signed-by=/usr/share/keyrings/Unity_Technologies_ApS.gpg] https: //hub.unity3d.com/linux/repos/deb stable main
  Active apt repos in: /etc/apt/sources.list.d/vscode.list
    1: deb [arch=amd64,arm64,armhf] http: //packages.microsoft.com/repos/code stable main
  Active apt repos in: /etc/apt/sources.list.d/winehq-jammy.sources
    1: deb [arch=amd64 i386] https: //dl.winehq.org/wine-builds/ubuntu jammy main
  Active apt repos in: /etc/apt/sources.list.d/winehq-kinetic.sources
    1: deb [arch=amd64 i386] https: //dl.winehq.org/wine-builds/ubuntu kinetic main
Info:
  Processes: 451 Uptime: 2d 1h 11m Memory: 31.12 GiB used: 6.02 GiB (19.3%) Init: systemd v: 249
  runlevel: 5 Compilers: gcc: 11.4.0 alt: 11/12 clang: 14.0.0-1ubuntu1.1 Client: Cinnamon v: 5.8.4
  inxi: 3.3.13
```

**Comment by Stonley890** [comment]
Images
![Screenshot from 2023-11-17 17-52-19](https://github.com/Vencord/Vesktop/assets/79172597/6ac419f6-c1b0-49bf-bd38-cd6918de3f74)
![Screenshot from 2023-11-17 17-54-23](https://github.com/Vencord/Vesktop/assets/79172597/818b54a9-945d-4835-be06-0ebf641d3bd5)

**Comment by Curve** [comment]
Can you please also share the console output when starting Vesktop from the terminal?

**Comment by Curve** [comment]
Okay this is kinda funny, by lowering the GLIBCXX version requirement we accidentally increased the GLIBC minimum required version.

How should we proceed with this @Vendicated ? I think it's more realistic for a distro to have a more recent libstdc++ version than for it to have a newer glibc version actually

**Comment by Curve** [comment]
Fixed: in Vencord/venmic@6c356f688450fd7f6f5a5a4f26dd89be6fea0238
